packages/angular/src/list/list-item/ai-list-item.component.ts

```diff
@@ -78,6 +78,7 @@ import { IconService } from 'carbon-components-angular';
             [ngClass]="{ 'iot--list-item--content--values--main__large': item.size === 'lg' }"
           >
             <div
+              title="Item {{ item.value }}"
```

Comment:
I would imagine you would want this to come from a translatable string.

Reply:
yes , It's fine  and I removed the hardcoded value `Item`  .